### PR TITLE
Message is properly aligned when sarched skill is not found

### LINKED ISSUE #3093: Improper alignment of message when "no skill is found"
**Actual Behaviour**

When a user searches for a skill and the skill with the name searched is not found then following message is displayed.

Screenshot (8)

We can see that the alignment of the 2 bulletted points is not correct.

**Expected Behaviour**

The 2 bulletted points should be aligned properly under the heading like this:

Screenshot (7)

**Steps to reproduce it**

The problem is due to the inheritance property of the 'text-align' property in CSS.
The sub-elements i.e. the bullet points are also inheriting the text-align:center property and are being displayed in center according to the user's window

**Screenshots of the issue**

Added above for better understanding

**Would you like to work on the issue?**

Yes, would definately like to work on this issue

### PR DESCRIPTION
**Pull request by paras151**
Fixes #3093 

Changes: Now the bullet points in message are directly under the heading and properly aligned

Demo Link: https://pr-3080-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot (9)

### CODE REVIEW

File: src/components/cms/BrowseSkill/BrowseSkill.js [hunk 1]
```diff
@@ -677,15 +677,25 @@ class BrowseSkill extends React.Component {
...
-            <li>Using more general terms</li>
-            <li>Checking your spelling</li>
-          </ul>
+          <div style={{ display: 'flex', flexDirection: 'column' }}>
+            <h2 style={{ fontWeight: '400' }}>
+              Your search <b>&quot;{searchQuery}&quot;</b> did not match any
+              skills.
+            </h2>
+            <div
+              style={{
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/BrowseSkill.js]: Use styled-components for styling

Comment (paras151) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: ok, are you referring to the div(line 680) that I have created and want these changes using styled-components.

**Comment by Dishebh** [comment]
@paras151 any progress uptil now?

File: src/components/cms/BrowseSkill/BrowseSkill.js (hunk 1 above)
Comment (akshatnitd) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: @paras151 Yes

File: src/components/cms/BrowseSkill/BrowseSkill.js [hunk 2]
```diff
@@ -677,15 +677,25 @@ class BrowseSkill extends React.Component {
...
-          <h2 style={{ fontWeight: '400' }}>
-            Your search <b>&quot;{searchQuery}&quot;</b> did not match any
-            skills.
-          </h2>
-          <h3 style={{ margin: '15px 0 10px 0' }}>Try something like</h3>
-          <ul style={{ listStyle: 'inside' }}>
-            <li>Using more general terms</li>
-            <li>Checking your spelling</li>
-          </ul>
+          <div style={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (samagragupta) [inline src/components/cms/BrowseSkill/BrowseSkill.js]: Use styled-components

**Comment by akshatnitd** [comment]
Fixed in https://github.com/fossasia/susi.ai/pull/3538. Hence, closing.

**Pull request closed without merging**